ONSdigital/rasrm-acceptance-tests
# Alterations for us023

**Don't merge until response-ops branch `reenable-respondent-enrolment` is merged**

# Motivation and Context
US023 added the ability to re-enable enrolment for a repsondent, which meant introducing an extra link - changes test that re-enabling works.

Also added a change to allow screenshots for test diagnostics.

# What has changed
* Altered acceptance tests for US022 to include criterion for US023
* Added `environment.py` code to allow the tests to generate screenshot artifacts
* Set screen size to avoid `splinter` dynamically resizing the virtual window dependent on the contents

# How to test?
* Run the tests to confirm that they run successfully, once the `response-ops` branch has been merged.
* Set the `SCREENSHOT` environment variable to `True`, and force a test to fail.  Confirm that a screenshot is taken of the fail - note that some fails will generate a blank screenshot depending on the kind of fail.

# Links
* https://trello.com/c/StCt8X1m
